pkg/apisix/route.go

```diff
@@ -41,16 +41,24 @@ func newRouteClient(c *cluster) Route {
 }
 
 // Get returns the Route.
+func (r *routeClient) Get(ctx context.Context, name string) (*v1.Route, error) {
+	return r.get(ctx, id.GenID(name), name)
+}
+
+// GetByName returns the Route with route name
+func (r *routeClient) GetByID(ctx context.Context, realID string) (*v1.Route, error) {
+	return r.get(ctx, realID, realID)
+}
+
 // FIXME, currently if caller pass a non-existent resource, the Get always passes
 // through cache.
-func (r *routeClient) Get(ctx context.Context, name string) (*v1.Route, error) {
+func (r *routeClient) get(ctx context.Context, id string, name string) (*v1.Route, error) {
 	log.Debugw("try to look up route",
 		zap.String("name", name),
 		zap.String("url", r.url),
 		zap.String("cluster", "default"),
```

Comment:
Should also print the route id in logs.

Reply:
`GetById` also pass id as `name` in `get`. 
So changed `zap.String("name", name)` to `zap.String("name/id", name)`

Reply:
Here you don't change.

Reply:
🤣 My bad. Fixed